aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    butler_repo = '/sdf/group/rubin/u/brycek/aos-repos/aosTestDataRepoImsim/'
</pre><p>Did you find that this is the most persisted location? Or  would it work to create some sort of  eg. <code>/sdf/group/rubin/shared/aos</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I think one of the outcomes of our meeting discussion about shared butler space was that for smaller datasets this would be okay. Were you able to access the butler located there? Just want to make sure the permissions are set correctly.</p>

Reply:
<p>ok, I can access </p><p> </p><pre class="ql-syntax" spellcheck="false">/sdf/group/rubin/u/brycek/aos-repos/aosTestDataRepoImsim/
</pre><p> </p>

Reply:
<p>I can also access <code>/sdf/group/rubin/repo/aos_imsim/</code></p>